src/main/java/org/apache/accumulo/testing/performance/tests/ConditionalMutationsPT.java

```diff
@@ -101,7 +101,7 @@ private static void runConditionalMutationsTest(Environment env, String tableNam
     }
 
     reportBuilder.result("avgRate: 1-19",
-        Double.parseDouble(new DecimalFormat("#0.00").format(rateSum / 20)),
+        Double.parseDouble(new DecimalFormat("#0.00").format(rateSum / 19)),
```

Comment:
> It's weird that we run it 20 times, once before the loop of 19, but not collecting the sum for that first run.

I suspect the first run is done to warm up the JVMs (like make the JIT kick in) and its time is ignored.  Then the times from the next 19 runs are used.

Reply:
That seems reasonable. Do we do that for other Performance Tests?

Reply:
Warming up seems reasonable. I just don't know why we have 19 subsequent runs. It might make sense to just round the number of runs off to 20, rather than use 19 as a divisor, which is a little confusing. The warm-up run would benefit from an inline comment like `// warm-up run`

Reply:
I'm not sure why the first set only runs 19 times, I just thought I would fix the average calculation. I can go through and bump all the numbers up by 1 so that all runs are performed 20 times for consistency.

Reply:
Addressed in dc41ea0